executor/split_test.go

```diff
@@ -173,15 +166,15 @@ func (s *testSplitIndex) TestSplitIndex(c *C) {
 	for _, ca := range cases {
 		// test for minInt64 handle
 		idxValue, _, err := index.GenIndexKey(ctx.GetSessionVars().StmtCtx, []types.Datum{types.NewDatum(ca.value)}, kv.IntHandle(math.MinInt64), nil)
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		idx := searchLessEqualIdx(valueList, idxValue)
-		c.Assert(idx, Equals, ca.lessEqualIdx, Commentf("%#v", ca))
+		require.Equal(t, idx, ca.lessEqualIdx, ca)
 
 		// Test for max int64 handle.
 		idxValue, _, err = index.GenIndexKey(ctx.GetSessionVars().StmtCtx, []types.Datum{types.NewDatum(ca.value)}, kv.IntHandle(math.MaxInt64), nil)
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		idx = searchLessEqualIdx(valueList, idxValue)
-		c.Assert(idx, Equals, ca.lessEqualIdx, Commentf("%#v", ca))
+		require.Equal(t, idx, ca.lessEqualIdx, ca)
```

Comment:
require.Equal(t, idx, ca.lessEqualIdx)

Reply:
So I need to remove argument `ca` in this file? I am not clear about it's use actually :(